scalability_and_performance/node-observability-operator.adoc

```diff
@@ -20,6 +22,14 @@ include::modules/node-observability-install-cli.adoc[leveloffset=+2]
 
 include::modules/node-observability-install-web-console.adoc[leveloffset=+2]
 
-include::modules/node-observability-create-custom-resource.adoc[leveloffset=+1]
+== Requesting CRI-O and Kubelet profiling data using the Node Observability Operator
+
+include::modules/node-observability-create-custom-resource.adoc[leveloffset=+2]
+
+include::modules/node-observability-run-profiling-query.adoc[leveloffset=+2]
+
+== Node Observability Operator Scripting
```

Comment:
This also needs to begin with a gerund since both sub-sections are procedural modules (similar to installing.. and Requesting...)
```suggestion
== Node Observability Operator scripting
```

Reply:
Let me change it.